.github/workflows/vscodeSupportE2E.yml

```diff
@@ -0,0 +1,111 @@
+name: VS Code Support E2E Tests
+on:
+  workflow_run:
+    workflows:
+      - Nightly Build Develop
+    types:
+      - completed
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+      testToRun:
+        description: 'Run this E2E test'
+        required: false
+        default: 'anInitialSuite.e2e.ts'
+        type: string
+      runId:
+        description: 'Run ID of the workflow run that created the vsixes'
+        required: false
+        type: string
+      vscodeVersion:
+        description: 'VS Code Version'
+        required: false
+        type: string
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        nodeVersion:
+          - 18.11.0
+        vscodeVersion:
+          - ${{ inputs.vscodeVersion || '1.61.2'}}
```

Comment:
According to the VSCode documentation, I see that 1.61.2 is the VSCode version from September 2021 and a new minor version is released every month.  Since we are going with the hard-coding the VSCode version for now, this does mean that we need to update this workflow once a month, right?